code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -154,11 +199,13 @@ Implant Specifics:<BR>"}
 	var/elevel = "Localized Limb"
 	var/phrase
 	var/setup_done = FALSE //Have we set this yet?
+	var/uses_codewords = TRUE 
 	icon_state = "implant_evil"
 
 /obj/item/implant/explosive/Initialize()
-	. = ..()
-	phrase = "You are [pick(adjectives)]"
+	if(uses_codewords)
+		. = ..()
+		phrase = "You are [pick(adjectives)]"
```

Comment:
```suggestion
	. = ..()
	if(uses_codewords)
		phrase = "You are [pick(adjectives)]"
```
You always are going to want to . = ..() on initialize. You might get buggy behavior otherwise.